ProjectTemplates/VisualStudio2010/DesktopGL/MonoGameDesktopGL.vstemplate

```diff
@@ -1,7 +1,7 @@
 <VSTemplate Version="3.0.0" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005" Type="Project">
   <TemplateData>
-    <Name>MonoGame Linux Project</Name>
-    <Description>A MonoGame game project for the Linux desktop.</Description>
+    <Name>MonoGame OpenGL Project</Name>
+    <Description>A MonoGame game project for Windows, Mac and Linux using OpenGL.</Description>
```

Comment:
I named templates like that because MonoDevelops/Xamarin Studios templates are named "MonoGame Application (OpenGL)".

Out of the 4 names you mentioned:
 - "MonoGame Desktop Project" - there is also WindowsDX, kind of confusing
 - "MonoGame Universal Desktop Project" - confusing because of UAP/UWP
 - "MonoGame Cross Platform Desktop Project" - sounds ok
 - "MonoGame Win/Mac/Linux Desktop Project" - mouthful